rskj-core/src/main/java/co/rsk/config/BridgeConstants.java

```diff
@@ -128,9 +128,15 @@ public long getFundsMigrationAgeSinceActivationBegin() {
     }
 
     public long getFundsMigrationAgeSinceActivationEnd(ActivationConfig.ForBlock activations) {
-        return activations.isActive(ConsensusRule.RSKIP357) ?
-            specialCaseFundsMigrationAgeSinceActivationEnd :
-            fundsMigrationAgeSinceActivationEnd;
+        if (activations.isActive(ConsensusRule.RSKIP374)){
+            return fundsMigrationAgeSinceActivationEnd;
+        }
+
+        if (activations.isActive(ConsensusRule.RSKIP357)){
+            return specialCaseFundsMigrationAgeSinceActivationEnd;
+        }
```

Comment:
```suggestion
        if (activations.isActive(ConsensusRule.RSKIP357) && !activations.isActive(ConsensusRule.RSKIP374)){
            return specialCaseFundsMigrationAgeSinceActivationEnd;
        }
```
I think that simplifies it